src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -190,37 +197,8 @@ public Integer call() {
                 useNISTRestriction ? ValidateAIF.Restriction.NIST : ValidateAIF.Restriction.NONE;
         final boolean profiling = useProfiling || useProgressiveProfiling;
 
-        // Finally, try to create the validator, but fail if required elements can't be loaded/parsed.
```

Comment:
Why move this until after the files are collected?

Reply:
I didn't want to create the `ValidateAIF` object (and its potentially disk-based model) if there was a chance we'd get a file error because no ttls were specified (either due to mistyping a file name or not having and ttls in a directory, etc.).  Or really any other error...basically I didn't want to create the disk-based model until we'd gotten through as many error conditions as possible.

Reply:
Sounds good to me. Thanks